test/data/github.push.json

```diff
@@ -142,21 +142,21 @@
     "email": "baxterthehacker@users.noreply.github.com"
   },
   "sender": {
-    "login": "baxterthehacker",
+    "login": "baxterthehacker2",
     "id": 6752317,
```

Comment:
I'm not going to call out each instance. Just make sure to update the `id` of the `sender` and the `avatar_url` across all the payloads